### PR TITLE
[KOGITO-2004] - Default the Quarkus S2I image to build non native images

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Jira Issue : https://issues.redhat.com/browse/KOGITO-2004

### CODE REVIEW

File: modules/kogito-quarkus-s2i/s2i/bin/usage [hunk 1]
```diff
@@ -20,12 +20,19 @@ Sample invocation:
 $ s2i build -c /Development/project/ quay.io/kiegroup/kogito-quarkus-ubi8-s2i <application image> --runtime-image quay.io/kiegroup/kogito-quarkus-ubi8
 
 
-For non native build, just set the NATIVE environment variable to false and use the quay.io/kiegroup/kogito-quarkus-jvm-ubi8
-image as runtime-image, in that way a *runner.jar will be produced and copied to the runtime image.
+By default it generate non native build. Use the quay.io/kiegroup/kogito-quarkus-jvm-ubi8 image as runtime-image,
```
Comment (radtriste) [inline modules/kogito-quarkus-s2i/s2i/bin/usage]:
```suggestion
By default it generates non native build. Use the quay.io/kiegroup/kogito-quarkus-jvm-ubi8 image as runtime-image,
```

**Review by radtriste — CHANGES_REQUESTED** [review]
small change, otherwise looks good.

**Review by spolti — APPROVED** [review]
nice work.
I'll approve but before merge address Tristan suggestions.

**Comment by spolti** [comment]
@vaibhavjainwiz please take a look on the CI, this feature is failing:

```
  features/image/kogito-quarkus-ubi8-s2i.feature:59  Verify if the s2i build is finished as expected performing a non native build with runtime image
```

File: modules/kogito-quarkus-s2i/s2i/bin/usage (hunk 1 above)
Comment (vaibhavjainwiz) [reply modules/kogito-quarkus-s2i/s2i/bin/usage]: done

**Comment by spolti** [comment]
@vaibhavjainwiz CI still failing, can you please check it?

**Comment by vaibhavjainwiz** [comment]
> @vaibhavjainwiz CI still failing, can you please check it?

@spolti  Jenkin builds was failed due to some Jenkins issues. It passed now after retrigger

**Review by radtriste — APPROVED** [review]
lgtm

**Pull request merged**